Ports/stress-ng/patches/ReadMe.md

```diff
@@ -0,0 +1,41 @@
+# Patches for stress-ng
+
+## `0010-disable-rand48-cpu-stressor.patch`
+FIXME: This patch should be documented, and possibly split into multiple atomic patches.
```

Comment:
Looks like I missed at least this one, I'll make that script and update this...not today though.